src/components/Property/RaiseWorkOrder/RaiseWorkOrderForm.js

```diff
@@ -28,32 +28,32 @@ const RaiseWorkOrderForm = ({
   onFormSubmit,
   raiseLimit,
 }) => {
-  const { register, handleSubmit, errors } = useForm()
+  const { register, handleSubmit, errors, setValue } = useForm()
   const [priorityCode, setPriorityCode] = useState('')
-  const priorityList = priorities.map((priority) => priority.description)
 
   const [totalCost, setTotalCost] = useState('')
   const overSpendLimit = totalCost > raiseLimit
 
   const onSubmit = async (formData) => {
-    const scheduleWorkOrderFormData = buildScheduleWorkOrderFormData(formData)
+    const scheduleWorkOrderFormData = buildScheduleWorkOrderFormData({
+      ...formData,
+      priorityDescription: getPriorityObjectByCode(formData.priorityCode)
+        .description,
```

Comment:
Needed because the form no longer has any field that contains the description - it's derived from the code, now.